lib/book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+
+  attr_reader :author_first_name, :author_last_name, :author, :title, :publication_date, :publication_year
```

Comment:
Since we only want to be able to call `.author`, `.title`, and `.publication_year`, these are the only instance variables we need - the others are overkill